Marlin/ultralcd.cpp

```diff
@@ -1096,7 +1227,9 @@ void kill_screen(const char* lcd_msg) {
         const int32_t rounded = (int32_t)(mesh_edit_value * 1000.0);
         mesh_edit_value = float(rounded - (rounded % 5L)) / 1000.0;
       }
-
+#if ENABLED(AUTO_BED_LEVELING_UBL_GFX)
```

Comment:
This needs to be inside the "if(lcdDrawUpdate)", after "lcd_implementation_drawedit". The reason being that if it isn't, the graphics may be overwritten.